src/InsertionsClient.Console/ConsoleApp/Program.cs

```diff
@@ -50,6 +52,8 @@ internal class Program
             txt.Append(" ");
             txt.Append($"{SwitchManifest}<manifest.json full file path>");
             txt.Append(" ");
+            txt.Append($"{SwitchBuildFilter}<filter that selects builds to insert>");
```

Comment:
minor perf comment:  the # of appends has significantly grown since the project's early days.  Consider a different StringBuilder .ctor.  
https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.-ctor?view=netcore-3.1#System_Text_StringBuilder__ctor_System_Int32_ 

StringBuilder does some magic to grow its' buffer as needed.  Look at the IL if you want to know more.

Reply:
Looks like I accidentally merged before I could push all the changes. I will create a new PR for this.